code/test/e2e/loggedin/marketplace.spec.js

```diff
@@ -7,7 +7,7 @@ test('Marketplace shows only published apps', async ({ page }, { project, config
   await page.getByRole('link', { name: 'Apps' }).click();
   await expect(page).toHaveURL(marketplaceUrl);
 
-  const appCards = await page.locator('a.ui.card').nth(0).waitFor({ timeout: 5000 });
+  const appCards = await page.locator('a.ui.card').nth(0).waitFor({ timeout: 20_000 });
```

Comment:
is underscore supported `20_000`?

Reply:
Yes, JavaScript and Java support it, but not Clojure